tests/test_smart_proxy.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+
+
+@pytest.fixture
+def content_smart_proxy(api, organization):
+    smart_proxies = api.resource('smart_proxies').call('index', {'sort_by': 'id', 'sort_order': 'DESC'})
+    content_proxy = smart_proxies['results'][0]
+    content_proxy_id = content_proxy['id']
+
+    environments = api.resource('capsule_content').call('lifecycle_environments', {
+        'id': content_proxy['id']
+    })['results']
+
+    for environment in environments:
+        api.resource('capsule_content').call('remove_lifecycle_environment', {
+            'id': content_proxy_id,
+            'environment_id': environment['id']
+        })
+
+    library = api.resource('lifecycle_environments').call('index', {
+        'search': 'Library',
+        'organization_id': organization['id']
+    })['results'][0]
+
+    api.resource('capsule_content').call('add_lifecycle_environment', {
+        'id': content_proxy_id,
+        'environment_id': library['id']
+    })
+
+    return content_proxy
+
+
+@pytest.fixture
+def organization(api, entities):
+    return api.resource('organizations').call('index', {
+        'search': entities['organization_label']
+    })['results'][0]
+
+
+def test_smart_proxy_content_sync(api, content_smart_proxy):
+    assert api.resource('capsule_content').call('sync', {'id': content_smart_proxy['id']})
```

Comment:
Is this a sync task or do we need to wait for it to complete? Can we somehow verify it actually synced?

Reply:
I'll update this to check the task completes. Over in bats, the content is checked, I just had not moved that over yet:

https://github.com/theforeman/forklift/blob/master/bats/fb-katello-proxy.bats#L29